src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -200,7 +200,7 @@ private void LoadCommitsTime(Docset docset)
         {
             if (!string.IsNullOrEmpty(docset.Config.Contribution.GitCommitsTime))
             {
-                var (_, content, _) = RestoreMap.GetRestoredFileContent(docset, docset.Config.Contribution.GitCommitsTime);
+                var (_, content, _) = RestoreMap.GetRestoredFileContent(docset, docset.Config.Contribution.GitCommitsTime, docset.Config.Contribution.GitCommitsTime.Range);
```

Comment:
Add an overload `GetRestoredFileContent(Docset docset, SourceInfo<string> url)`? Seems it needn't 3 params. Same for `DependencyLock.Load`

Reply:
updated